pipeline/config/exoscale/common-config.yaml

```diff
@@ -54,3 +54,10 @@ networkPolicies:
     alertReceivers:
       ips:
         - 0.0.0.0/0
+  coredns:
+    externalDns:
+      ips:
+        - 0.0.0.0/0
+    serviceIp:
+      ips:
+        - 0.0.0.0/0
```

Comment:
Why are you not using the default value for serviceIp here?

Reply:
I had it initially set to the default `10.233.0.3/32` but then we thought that this might differ on some cloud providers and decided to use the `0.0.0.0/0` instead and we can set it in the configs to the actual value.